Answers.md

```diff
@@ -1,9 +1,20 @@
 1. If you had to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic html is html that is written to convey meaning about the content of the html and not just its presentational aspects.
+
 2. Describe some differences between ```display: block;``` and ```display: inline;```.
 
+"display: inline" is mostly for tags used within other elements, such as a <p>.  You cannot set a height, width, or vertical margin on an inline element.
+"display: block" can have a width, height, and margin.
```

Comment:
I would add the display:block normally causes a line break before the next element